.github/ISSUE_TEMPLATE/improvement_request.yml

```diff
@@ -0,0 +1,46 @@
+name: "📈 Improvement request"
```

Comment:
i don't want to split

Reply:
+1, there are many instances where this line could be blurry or change, forcing a split at issue creation might result in more labeling/re-labeling work later on (which could lead to issues getting lost) @Jay-o-Way

Reply:
Would a checlbox "Improvement for an existing feature" be the better choice?